extension/js/common/message-renderer.ts

```diff
@@ -431,7 +431,10 @@ export class MessageRenderer {
 
   private getMessageInfo = async (fullMsg: GmailRes.GmailMsg): Promise<MessageInfo> => {
     const sentDate = GmailParser.findHeader(fullMsg, 'date');
-    const sentDateStr = sentDate ? Str.fromDate(new Date(sentDate)).replace(' ', ' at ') : '';
+    const gmailDateReceived = $('div.gK span[title]').attr('title');
+    // detects current timezone by checking if Gmail's dateTime includes trailing AM/PM otherwise its 24hour format
+    const ishour12Format = (gmailDateReceived !== undefined && gmailDateReceived.endsWith('M') ? true : false) || false;
```

Comment:
it can be simplified to `const isHour12Format = gmailDateReceived?.endsWith('M')`

Reply:
thanks!